llama_index/evaluation/tonic_validate/answer_consistency.py

```diff
@@ -0,0 +1,66 @@
+from typing import Any, Optional, Sequence
+
+from llama_index.evaluation.base import BaseEvaluator, EvaluationResult
+from llama_index.prompts.mixin import PromptDictType, PromptMixinType
+
+
+class AnswerConsistencyEvaluator(BaseEvaluator):
+    """Tonic Validate's answer consistency metric.
```

Comment:
NIT: can we add the score range in the doc strings for this evaluator and all other ones as well?

And, related to that, does Tonic do any validations on the score values?

Reply:
I just pushed a change to add the score ranges to the doc strings.

We have validation to check that the scores returned from the llm-judge can be parsed as floats. If they cannot, then an error message is logged and a score of 0.0 is returned. We do not have validation that the scores fall into the range for which they are supposed to though. Thanks for this comment, I made an issue in the `tonic_validate` repo to add this validation.

Reply:
Cool. Thanks for this information :)